CHANGELOG.md

```diff
@@ -1,8 +1,83 @@
+## 1.1.4
```

Comment:
The changes listed here are not 1.1.4, they're 1.2.0. There is also a 1.1.4 release.